pages/content/reports/2024/index.md

```diff
@@ -0,0 +1,232 @@
+
+| Project                 | Total authored PRs | OWD authored PRs |
+| ----------------------- | ------------------ | ---------------- |
+| mdn/content             | 3907               | 551 (14.1%)      |
+| mdn/browser-compat-data | 2034               | 1066 (52.4%)     |
+| webdx/web-features      | 1150               | 170 (14.8%)      |
+
+### Pull requests reviewed
+
+As in previous years, there's a very long tail of contributors and a thriving community actively involved with our work. For mdn/content, we group all contributors into three categories: one-timers, casual contributors (2-9 PRs) and core contributors (10+ PRs). The 2024 breakdown looks like this:
+
+- 708 contributors had 1 PR merged.
+- 166 contributors had 2-9 PRs merged.
+- 44 contributors had 10 or more PRs merged.
+
+Supporting this long tail of volunteers is a large part of OWD's work. PR reviews for the mdn/content repository are performed by members of the maintainer group, which consists of OWD staff, Mozilla, Google and Microsoft staff, W3C staff, and a select group of volunteer maintainers.
+
+Special shout outs to volunteer reviewer [Josh-Cena](https://github.com/Josh-Cena), who alone took on 18.3% of reviews in 2024 for mdn/content, and to [Daniel D. Beck](https://github.com/ddbeck) who reviewed more than half (56.4%) of all web-features pull requests in 2024. A pleasure to work with both of you and your impact in 2024 on these two projects is very noteworthy! Thank you!
+
+Here's the summary of reviewed PRs:
+
+| Project                 | Total reviewed PRs | OWD reviewed PRs |
+| ----------------------- | ------------------ | ---------------- |
+| mdn/content             | 4595               | 1254 (27.3%)     |
+| mdn/browser-compat-data | 2209               | 1154 (52.3%)     |
+| webdx/web-features      | 1961               | 135 (6.9%)       |
+
+## Sovereign Tech Fund
+
+Open Web Docs is very proud to have received a second round investment from the Sovereign Tech Fund at the beginning of 2024. The [Sovereign Tech Fund (STF)](https://www.sovereign.tech/programs/fund) supports the development, improvement, and maintenance of open digital infrastructure in the public interest. The [Sovereign Tech Fund’s Contribute Back Challenges](https://www.sovereign.tech/programs/challenges) selected Open Web Docs as a partner to implement improved tooling and maintenance of the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data) and further development of the [openwebdocs/mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) tool.
+
+As part of this program, that continued from the first round in 2023, we completed key milestones:
+
+- Open Web Docs (OWD) automated browser compatibility data updates using OWD’s [mdn-bcd-collector](https://github.com/openwebdocs/mdn-bcd-collector) project. Now, browser compatibility information is available to web developers for new browser releases faster than ever.
```

Comment:
re faster than ever, can we be more specific? (e.g. one week after the release on average?)